src/test/java/com/epam/tc/nitcenkov/hw9/TrelloApiTests.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.tc.nitcenkov.hw9;
+
+import com.epam.tc.nitcenkov.hw9.dto.BoardDTO;
+import com.epam.tc.nitcenkov.hw9.dto.ListDTO;
+import com.epam.tc.nitcenkov.hw9.service.BoardService;
+import com.epam.tc.nitcenkov.hw9.service.ListService;
+import org.testng.annotations.Test;
+
+public class TrelloApiTests extends BaseTests {
+
+    @Test(description = "Create board -> Get board by id")
+    public void createGetBoard() {
+        BoardDTO boardDto = new BoardService().createBoard(paramsForBoardRequests);
+        String boardId = boardDto.getId();
+        BoardDTO justCreatedBoard = new BoardService().getBoardById(boardId);
+        softAssertions.assertThat(justCreatedBoard.getName()).isEqualTo("board");
+    }
```

Comment:
instead of direct assertions in test cases use layer of Assertions

so, for example, you might have

BoardAssertions.assertBoardNameId(boardDto, "name")